src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -35,95 +33,1003 @@ const ActorInit En_Sw_InitVars = {
+void EnSw_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnSw* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0DE34.s")
+    SkelAnime_Update(&this->skelAnime);
+    func_80B0C9F0(this, globalCtx);
+    this->actionFunc(this, globalCtx);
+    func_80B0CBE8(this, globalCtx);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0DEA8.s")
+s32 EnSw_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dListP, Vec3f* pos, Vec3s* rot, void* thisx) {
+    Vec3f sp7C = { 1400.0f, -2600.0f, -800.0f };
+    Vec3f sp70 = { 1400.0f, -1600.0f, 0.0f };
+    Vec3f sp64 = { -1400.0f, -2600.0f, -800.0f };
+    Vec3f sp58 = { -1400.0f, -1600.0f, 0.0f };
+    Vec3f sp4C = { 0.0, 0.0f, -600.0f };
+    EnSw* this = THIS;
+    Vec3f sp3C = { 0.0f, 0.0f, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0DFFC.s")
+    OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_sw.c", 2084);
+    if (((this->actor.params & 0xE000) >> 0xD) != 0) {
+        switch (limbIndex) {
+            case 23:
+                *dListP = &D_06004788;
+                break;
+            case 8:
+                *dListP = &D_060046F0;
+                break;
+            case 14:
+                *dListP = &D_06004658;
+                break;
+            case 11:
+                *dListP = &D_060045C0;
+                break;
+            case 26:
+                *dListP = &D_06004820;
+                break;
+            case 20:
+                *dListP = &D_060048B8;
+                break;
+            case 17:
+                *dListP = &D_06004950;
+                break;
+            case 29:
+                *dListP = &D_060049E8;
+                break;
+            case 5:
+                *dListP = &D_06003FB0;
+                break;
+            case 4:
+                *dListP = &D_060043D8;
+                break;
+        }
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0E314.s")
+    if (limbIndex == 1) {
+        Matrix_MultVec3f(&sp7C, &this->unk_454);
+        Matrix_MultVec3f(&sp70, &this->unk_460);
+        Matrix_MultVec3f(&sp64, &this->unk_46C);
+        Matrix_MultVec3f(&sp58, &this->unk_478);
+        Matrix_MultVec3f(&sp4C, &this->unk_484);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0E430.s")
+    if (limbIndex == 5) {
+        Matrix_MultVec3f(&sp3C, &this->actor.focus);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0E5E0.s")
+    if (limbIndex == 4) {
+        gDPSetEnvColor(POLY_OPA_DISP++, this->unk_1F4.r, this->unk_1F4.g, this->unk_1F4.b, 0);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sw/func_80B0E728.s")
+    Collider_UpdateSpheres(limbIndex, &this->collider);
+    CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_sw.c", 2145);
+    return 0;
```

Comment:
```suggestion
    return false;
```
OverrideLimbDraw returns a boolean.